egui_demo_lib/src/apps/demo/plot_demo.rs

```diff
@@ -74,7 +74,7 @@ impl LineDemo {
                 ui.checkbox(animate, "animate");
                 ui.checkbox(square, "square view")
```

Comment:
```suggestion
                ui.checkbox(animate, "Animate");
                ui.checkbox(square, "Square view")
```

let's keep it consistent